### PR TITLE
fix: Azure destination test looking for wrong directory

### PR DESCRIPTION
**Pull request by potter-potter**
Cherry pick Azure Destination fix from Roman's PR. The output test was looking in the wrong directory.

### CODE REVIEW

**Review by rbiseck3 — APPROVED** [review]

**Review by ryannikolaidis — CHANGES_REQUESTED** [review]
see comments

**Comment by ryannikolaidis** [comment]
make title of pr a conventional commit message (the pr title will be the merge commit title by merge queue)

**Comment by ryannikolaidis** [comment]
can you give a more clear description of what was wrong and how this fixes: both in changelog and pr description

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,9 +8,10 @@
 
 * **Add Weaviate destination connector** Weaviate connector added to ingest CLI.  Users may now use `unstructured-ingest` to write partitioned data from over 20 data sources (so far) to a Weaviate object collection.
 
-
 ### Fixes
 
+* **Fix Azure destination test directory** Solves a bug where Azure destination test was was looking in the wrong directory.
+
```
Comment (ryannikolaidis) [inline CHANGELOG.md:14]: Sorry, actually coming back to this, I think we typically omit changelog entries for changes that solely affect tests. So maybe revert this part altogether?

Comment (potter-potter) [reply CHANGELOG.md:14]: @ryannikolaidis So I can completely remove changelog and version from this PR?

Comment (rbiseck3) [reply CHANGELOG.md:14]: Changelog version still needs to be bumped, but the comment can be removed.

**Comment by potter-potter** [comment]
code added to lint improvement instead.

**Pull request closed without merging**